### PR TITLE
EPICENTER-2568: add test for run api variable with &

### PR DESCRIPTION
**Pull request by senluchen2015**
POST has the same arguments in the body as expected

### CODE REVIEW

File: tests/spec/test-run-api-service.js [hunk 1]
```diff
@@ -406,6 +406,29 @@
...
             });
+
+            it('should call GET with & in variable name', function () {
+                var rs = new RunService({ account: account, project: project });
+                rs.query({ '&saved': true, '.price&': '1' });
+                var req = server.requests.pop();
+                req.url.should.equal(baseURL + ';&saved=true;.price&=1/');
+            });
+
+            it('should call GET with & in long variable names', function () {
```
Comment (narenranjit) [inline tests/spec/test-run-api-service.js]: I'd remove this test from here and add an equivalent version in test-variables-api-service.js. That should already have utility methods to do long GETS

File: tests/spec/test-run-api-service.js [hunk 2]
```diff
@@ -503,6 +526,31 @@
...
+
+            });
+            it('should do an GET with & in variable names', function () {
+                var rs = new RunService({ account: account, project: project });
+                rs.load('myfancyrunid', { include: ['score&', '&score2'] });
+                var req = server.requests.pop();
+                req.url.should.equal(baseURL + 'myfancyrunid/?include=score&,&score2');
+            });
+
+            it('should do an GET with & in long variable names', function () {
```
Comment (narenranjit) [inline tests/spec/test-run-api-service.js]: Similar to above, you can skip this test here, it only matters in the variables api service (unless I'm missing something this does)

File: tests/spec/test-run-api-service.js (hunk 1 above)
Comment (senluchen2015) [inline tests/spec/test-run-api-service.js]: @narenranjit are you saying that it should just test the variable api query()?

Comment (narenranjit) [inline tests/spec/test-run-api-service.js]: Yes - the specific issue the customer hit is that they had a variable with an ambersand which didn't work, so testing the variables service will be better.

**Comment by senluchen2015** [comment]
@narenranjit I moved it to variable api, and the split seems to work fine with '&' still.

**Pull request merged**